src/components/LoginForm/LoginForm.js

```diff
@@ -0,0 +1,120 @@
+import _ from 'lodash'
+import autobind from 'autobind-decorator'
+import cx from 'classnames'
+import React, { Component, PropTypes } from 'react'
+import { connect } from 'react-redux'
+import { bindActionCreators } from 'redux'
+import {
+  Button,
+  Field,
+  Form,
+  Input,
+  List,
+  ListItem,
+  Message,
+  Segment,
+} from 'stardust'
+
+import { login } from 'services/session'
+
+const FORM_SETTINGS = {
+  // stop Semantic UI from submitting form
+  onSuccess: () => false,
+}
+
+export default class LoginForm extends Component {
+  static propTypes = {
+    actions: PropTypes.object.isRequired,
+    className: PropTypes.string,
+    session: PropTypes.object.isRequired,
+  };
+
+  @autobind
+  handleSubmit(e) {
+    e.preventDefault()
+    if (!this.isFormValid()) return
+    const { email, password } = this.getCredentials()
+    this.props.actions.login(email, password)
+  }
+
+  @autobind
+  getCredentials() {
+    const { email, password } = this.refs.form.plugin('get values')
```

Comment:
This is a Stardust thing.  Stardust is built on Semantic UI, a frontend framework like Bootstrap.  Semantic UI uses jquery plugins for things like forms, dropdowns, etc.  Stardust exposes those plugins when you use the `ref` attribute.  See here, http://semantic-ui.com/behaviors/form.html#getting-values

In the long term, Stardust will get away from exposing the jQuery plugins and provide a more React way of doing this.